libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -893,6 +893,31 @@ void RC_Channel::do_aux_function_mission_reset(const AuxSwitchPos ch_flag)
     mission->reset();
 }
 
+bool RC_Channel::run_aux_function(aux_func_t ch_option, AuxSwitchPos pos, AuxFuncTriggerSource source) {
```

Comment:
{ on next line

Reply:
Congratulation!  You're the proud winner of 127 `const` review comments :-P

Change made.